Source/SIMPLib/Filtering/FilterPipeline.cpp

```diff
@@ -273,8 +313,8 @@ void FilterPipeline::fromJson(const QJsonObject& json, IObserver* obs)
                                  Possible reasons include a name change of the filter, plugin not loading or a simple spelling mistake? A \
                                                                                                                                         blank filter has been inserted in its place. Possible error message is: %1")
                          .arg(filterName);
-        PipelineMessage pm(filterName, ss, -66066, PipelineMessage::MessageType::Error);
-        pm.setPrefix("JsonFilterParametersReader::ReadPipelineFromFile()");
+        QString prefix = "JsonFilterParametersReader::ReadPipelineFromFile()";
+        PipelineErrorMessage::Pointer pm = PipelineErrorMessage::New(getName(), ss, -66066);
         obs->processPipelineMessage(pm);
```

Comment:
The QString prefix variable is never used.

Reply:
Also, how did that become a variable?

Reply:
It's a prefix that was leftover after removing the prefix as a message variable.  There are one or two places where we create the message ourselves and pass it into the processPipelineMessage method of an observer that we have access to.  Should probably change that to just use the setErrorCondition method and hook it up so that the observer's slot automatically gets called.